### PR TITLE
Added Google+ and LinkedIN Share for blog post

### PR DESCRIPTION
**Pull request by chipset95**
Added Google Plus share button using Google+ platform for web js.

### CODE REVIEW

**Comment by chipset95** [comment]
Build is failing but the buttons seem to be working.

Added Google Plus share button using Google+ platform for web js
Used [in.js](platform.linkedin.com/in.js) for LinkedIN share

**Comment by PanosSakkos** [comment]
Please have a look at the build failures, htmlproof is complaining about something :wink:

**Comment by chipset95** [comment]
```yml
- ./_site/tech/2015/06/05/404-page.html
  *  script is empty and has no src attribute (line 176)
- ./_site/tech/2015/06/10/contact-tutorial.html
  *  script is empty and has no src attribute (line 180)
- ./_site/tech/2015/06/11/social-buttons-tutorial.html
  *  script is empty and has no src attribute (line 183)
- ./_site/tech/2015/06/12/timeline-tutorial.html
  *  script is empty and has no src attribute (line 188)
- ./_site/tech/2015/06/15/rss-tutorial.html
  *  script is empty and has no src attribute (line 168)
- ./_site/tech/2015/06/19/writing-posts.html
  *  script is empty and has no src attribute (line 208)
- ./_site/tech/2015/06/20/blog-tutorial.html
  *  script is empty and has no src attribute (line 200)
- ./_site/tech/2015/06/24/intro-layout-tutorial.html
  *  script is empty and has no src attribute (line 213)
- ./_site/tech/2015/06/25/html-head-tutorial.html
  *  script is empty and has no src attribute (line 205)
- ./_site/tech/2015/06/27/welcome.html
  *  script is empty and has no src attribute (line 173)
```

This started happening after I added the following in `share.html`
```javascript
<script src="//platform.linkedin.com/in.js" type="text/javascript"> lang: en_US</script>
<script type="IN/Share" data-url="{{site.url}}{{ site.baseurl}}{{page.url}}" data-counter="right"></script>
```

Here the second `<script>` tag does not have an src, and it does not need it. Now I'm not sure how to avoid that error. I could try adding an empty src but then I'm afraid of having too many commits (in case it doesn't work) just to test one thing on Travis (commit history wont look good). If you're fine with that, I'll give it a try.

**Comment by PanosSakkos** [comment]
Hmm I guess that htmlproof has a flag for ignoring the src in the script, maybe it's worth finding it.

// Don't worry about experimenting with multiple commits, since you can squash and then (force) push to the PR branch :smile:

**Comment by chipset95** [comment]
Alright, I'll try it. And yeah saw the source of html proofer. It does have a flag for no src in `<script>`

**Comment by chipset95** [comment]
@PanosSakkos done. The commit 0d54b71 is a rebase squashing the other commits.

File: _includes/share.html [hunk 1]
```diff
@@ -24,6 +24,18 @@
...
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <div class="g-plus" data-action="share" data-href="{{site.url}}{{ site.baseurl}}{{page.url}}" data-annotation="bubble"></div>
+  </div>
+  {% endif %}
+
+  {% if site.linkedin-share %}
+  <div style="float:left; padding: 0 5px; vertical-align:top; background-color: #0077B5; border-radius:3px">
+    <a style="color: #fff; vertical-align:middle; align: middle;" href="https://www.linkedin.com/cws/share?url={{site.url}}{{site.baseurl}}{{page.url}}&amp;title={{site.title}} | {{page.title}}&amp;lang=en_US" target="_blank"><i class="fa fa-linkedin"></i> | Share</a>
+  </div>
+  {% endif %}
```
Comment (PanosSakkos) [inline _includes/share.html]: Let's place te Linked In Share before G+, since it will be more common to share on Linked In than in G+ :wink:

Comment (PanosSakkos) [inline _includes/share.html]: Let's place the Linked In Share before G+, since it will be more common to share on Linked In than in G+ :wink:

**Comment by chipset95** [comment]
@PanosSakkos et voilà :smile:

**Comment by PanosSakkos** [comment]
Awesome! :+1:

**Comment by chipset95** [comment]
Thanks :)

**Pull request merged**